homeassistant/components/alexa/smart_home.py

```diff
@@ -6,39 +6,33 @@
 from uuid import uuid4
 
 from homeassistant.components import (
-    alert, automation, cover, fan, group, input_boolean, light, lock,
+    alert, automation, cover, climate, fan, group, input_boolean, light, lock,
     media_player, scene, script, switch, http, sensor)
 import homeassistant.core as ha
 import homeassistant.util.color as color_util
+from homeassistant.util.temperature import convert as convert_temperature
 from homeassistant.util.decorator import Registry
 from homeassistant.const import (
-    ATTR_ENTITY_ID, ATTR_SUPPORTED_FEATURES, CONF_NAME, SERVICE_LOCK,
-    SERVICE_MEDIA_NEXT_TRACK, SERVICE_MEDIA_PAUSE, SERVICE_MEDIA_PLAY,
-    SERVICE_MEDIA_PREVIOUS_TRACK, SERVICE_MEDIA_STOP,
+    ATTR_ENTITY_ID, ATTR_SUPPORTED_FEATURES, ATTR_TEMPERATURE, CONF_NAME,
+    SERVICE_LOCK, SERVICE_MEDIA_NEXT_TRACK, SERVICE_MEDIA_PAUSE,
+    SERVICE_MEDIA_PLAY, SERVICE_MEDIA_PREVIOUS_TRACK, SERVICE_MEDIA_STOP,
     SERVICE_SET_COVER_POSITION, SERVICE_TURN_OFF, SERVICE_TURN_ON,
     SERVICE_UNLOCK, SERVICE_VOLUME_SET, TEMP_FAHRENHEIT, TEMP_CELSIUS,
     CONF_UNIT_OF_MEASUREMENT, STATE_LOCKED, STATE_UNLOCKED, STATE_ON)
-from .const import CONF_FILTER, CONF_ENTITY_CONFIG
+from .const import (
+    CONF_FILTER, CONF_ENTITY_CONFIG, CONF_DESCRIPTION,
+    CONF_DISPLAY_CATEGORIES, API_DIRECTIVE, API_ENDPOINT, API_EVENT,
+    API_CONTEXT, API_HEADER, API_PAYLOAD, SMART_HOME_HTTP_ENDPOINT,
+    CLIMATE_SUPPORTED_MODES)
 
 _LOGGER = logging.getLogger(__name__)
 
-API_DIRECTIVE = 'directive'
-API_ENDPOINT = 'endpoint'
-API_EVENT = 'event'
-API_CONTEXT = 'context'
-API_HEADER = 'header'
-API_PAYLOAD = 'payload'
-
 API_TEMP_UNITS = {
     TEMP_FAHRENHEIT: 'FAHRENHEIT',
     TEMP_CELSIUS: 'CELSIUS',
+    'K': 'KELVIN',
```

Comment:
When can this happen?

Reply:
The mapping for 'K' isn't used, but I added it to the dictionary for completeness.

Reply:
We should only include code that is being used.